domain-models/samples/AU128.sosa.ttl

```diff
@@ -1,28 +1,33 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix foiType: <http://pid.geoscience.gov.au/def/voc/featureofinteresttype/> ;
+@prefix foiType: <http://pid.geoscience.gov.au/def/voc/featureofinteresttype/> .
 @prefix geo: <http://www.opengis.net/ont/geosparql#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix sosa: <http://www.w3.org/ns/sosa/> .
 @prefix tern: <https://w3id.org/tern/ontologies/tern/> .
 @prefix wiki: <https://www.wikidata.org/wiki/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-<http://pid.geoscience.gov.au/sample/AU128> 
+<http://pid.geoscience.gov.au/sample/AU128>
     a sosa:Sample ;
     rdfs:label "Sample AU128" ;
     sosa:isSampleOf <http://pid.geoscience.gov.au/site/17594> ;
     sosa:hasSampledFeature <http://sweetontology.net/realmGeol/Lithosphere> ;
+    sosa:isResultOf [
+        a sosa:Sampling ;
+        sosa:hasSamplingMethod <http://pid.geoscience.gov.au/def/voc/ga/igsncode/Rock> ;
+        sosa:resultTime "TODO: Unclear in data which datetime is relevant" ;
```

Comment:
Let's keep even example RDF completely valid, so comment out the line as the TODO statement won't work as a date with `sosa:resultTime`:

```turtle
       # sosa:resultTime TODO: Unclear in data which datetime is relevant
```